distribution/docs/src/main/resources/content/_configuring/search-options.adoc

```diff
@@ -0,0 +1,23 @@
+:title: Configuring Search Options for ${catalog-ui}
+:type: subConfiguration
+:status: published
+:parent: Configuring ${catalog-ui}
+:order: 00
+:summary: Configuring local, historical, and archived Search Options for ${catalog-ui}.
+
+== {title}
+
+${catalog-ui} Gives the user a few options to control what metacards can be searched for.
+By default, the user can perform searches that produce historical metacards, archived metacards,
+and metacards from the local catalog. If the user wants to, however, they can disable searching
+for any of these types of metacards.
+
+.Configuring Search Options
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-ui}* application.
+. Select the *Configuration* tab.
+. Select Catalog UI Search
+. Scroll down until you find the "Disable Local Catalog" option with the other options below it.
```

Comment:
✏️ `Under the "Disable Local Catalog" option, check each metacard type for which to enable searching.`